fast64_internal/oot/oot_constants.py

```diff
@@ -1,3 +1,11 @@
+import os, xml.etree.ElementTree as ET
+from ..utility import PluginError
+
+# Read the XML file, throws an error if the file is missing
+try: tree = ET.parse(os.path.dirname(os.path.abspath(__file__)) + '/ActorList.xml')
+except: PluginError("ERROR: File 'fast64_internal/oot/ActorList.xml' is missing.")
```

Comment:
```suggestion
except: raise PluginError("ERROR: File 'fast64_internal/oot/ActorList.xml' is missing or malformed.")
```
I'm not sure what effect raising an error during module loading has, I'd expect prevent fast64 from loading but idk